frontend/.gitignore

```diff
@@ -0,0 +1,15 @@
+*.iml
+.gradle
+/local.properties
+/.idea/caches
```

Comment:
Not sure if the .idea folder should be committed, see here:
https://stackoverflow.com/questions/16736856/what-should-be-in-my-gitignore-for-an-android-studio-project

GitHub has a good .gitignore starter for Android and other projects too: https://github.com/github/gitignore/blob/main/Android.gitignore

Reply:
Thanks! I'll fix this up real quick